models/analytics/dimensions/activity_dim.sql

```diff
@@ -0,0 +1,25 @@
+{{ config(materialized='table', schema='gold', alias='activity_dim') }}
+
+WITH unique_activities AS (
+    SELECT
+        DISTINCT encounter_type AS activity_type
+    FROM intermediate.encounters_normalized
+),
+ranked_activities AS (
+    SELECT
+        ROW_NUMBER() OVER (ORDER BY activity_type) AS id,
+        activity_type
+    FROM unique_activities
+)
+SELECT
+    id,
+    activity_type,
+    -- location_id placeholder here - to be joined with another table
+    NULL AS location_id,
+    CURRENT_TIMESTAMP AS create_timestamp,
+    --  create_audit_id to be populated during time of load
+    NULL AS create_audit_id,
```

Comment:
Let's use invocation id or md5 of a NK/PK for this

Reply:
yes cool, once we know the NK/PK we can address this.

Reply:
Yeah for invocation id command - you can use this ` '{{ invocation_id }}' AS create_audit_id`

Reply:
gotcha, i will use that. thank you.